packages/smooth_app/lib/themes/color_schemes.dart

```diff
@@ -0,0 +1,65 @@
+import 'package:flutter/material.dart';
+
+const Color seed = Color(0xFF99460D);
+
+const ColorScheme lightColorScheme = ColorScheme(
+  brightness: Brightness.light,
+  primary: Color(0xFF99460D),
+  onPrimary: Color(0xFFFFFFFF),
```

Comment:
According to [figma](https://www.figma.com/file/nFMjewFAOa8c4ahtob7CAB/Mobile-App-Design-(Quentin)?node-id=2949%3A12841) the primary is white and the onPrimary is black.

Reply:
Here I was just taking the colors from the "color grid"?? from on top of the figma file, there onPrimary is white.

Just checked he is using surface as the color of the appBar
![grafik](https://user-images.githubusercontent.com/39344769/170822162-9941d12f-a8c6-4a61-9256-87016f5a0709.png)


Should we update the color scheme or update the Theme to use the Surface color as AppBar background

Reply:
Hi @M123-dev!
I don't know how Quentin named/selected his colors, but the standards in flutter are maybe a bit different.
If you have a look at the code of `AppBar`, this is what you read:
```dart
  /// If null, then the [AppBarTheme.backgroundColor] is used. If that value is also
  /// null, then [AppBar] uses the overall theme's [ColorScheme.primary] if the
  /// overall theme's brightness is [Brightness.light], and [ColorScheme.surface]
  /// if the overall theme's [brightness] is [Brightness.dark].
  /// ...
  final Color? backgroundColor;
```

No mention of "surface" then, even if of course in some cases the values can be similar.

I suggest to be as close as possible to the standards, in order to have a standard behavior of the app and a common language with all flutter developers.

Reply:
I'm currently looking for a way to make some kind of standard behaviour but honestly I'm kinda confused. There are sometimes even colors from the dark colorscheme in a light mock and it seems like `on-primary-container` got the primary color of the app on some points. There seems to be no overall logic in it. I'm writing a comment in figma